cfme/tests/configure/test_access_control.py

```diff
@@ -1001,3 +1001,41 @@ def test_tenant_quota_input_validate(appliance):
         view.form.fill({'{}_cb'.format(field[0]): True, '{}_txt'.format(field[0]): field[1]})
         assert view.save_button.disabled
         view.form.fill({'{}_cb'.format(field[0]): False})
+
+
+@pytest.fixture(scope='module')
+def first_level_tenant(appliance):
+    root_tenant = appliance.collections.tenants.get_root_tenant()
+    tenant = appliance.collections.tenants.create(name='Tenant_{}' + fauxfactory.gen_alphanumeric(),
```

Comment:
`Tenant_{}' + fauxfactory.gen_alphanumeric()` I believe a typo is here.

Reply:
Corrected.